jira/jira_tools/tools/basic_funcs.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+import requests
+from requests.exceptions import HTTPError
+
+ATLASSIAN_RESOURCES_URL = "https://api.atlassian.com/oauth/token/accessible-resources"
+ATLASSIAN_JIRA_API_URL = "https://api.atlassian.com/ex/jira"
+
+
+def get_jira_cloud_id() -> str:
+    token = os.getenv("JIRA_OAUTH_TOKEN", "")
```

Comment:
please add validation that token is not empty.
I know tool manager should not allow it but it's better to be safe + print error